tools/clang/test/hash_stability/lit.cfg

```diff
@@ -0,0 +1,42 @@
+# -*- Python -*-
```

Comment:
nit: None of the test directories are `_` separated names so this should be `HashStability`.